lib/CleantalkSP/SpbctWP/State.php

```diff
@@ -359,6 +366,8 @@ protected function init()
         $this->data['site_utc_offset_in_seconds'] = (current_time('timestamp') - time());
 
         $this->data['wl_company_name'] = $this->data["wl_mode_enabled"] ? $this->data["wl_brandname"] : $this->default_data['wl_company_name'];
+
+        $this->feature_restrictions = new FeatureRestriction\FeatureRestrictionService();
```

Comment:
It will be better to pass one time `$spbc` into `FeatureRestrictionService` object via parameter instead of passing `$spbc` every time into `getState` method via parameter.
Like this:
```php
new FeatureRestrictionService(this)
```